src/html/classic/ng/src/web/pages/extras/trashcanpage.js

```diff
@@ -0,0 +1,493 @@
+*
+* Authors:
+* Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+*
+* Copyright:
+* Copyright (C) 2017 Greenbone Networks GmbH
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+
+import _ from 'gmp/locale.js';
+
+import TrashActions from './trashactions.js';
+
+import Agent from 'gmp/models/agent.js';
+import Alert from 'gmp/models/alert.js';
+import Scanconfig from 'gmp/models/scanconfig.js';
+import Credential from 'gmp/models/credential.js';
+import Filter from 'gmp/models/filter.js';
+import Group from 'gmp/models/group.js';
+import Note from 'gmp/models/note.js';
+import Override from 'gmp/models/override.js';
+import Permission from 'gmp/models/permission.js';
+import PortList from 'gmp/models/portlist.js';
+import ReportFormat from 'gmp/models/reportformat.js';
+import Role from 'gmp/models/role.js';
+import Scanner from 'gmp/models/scanner.js';
+import Schedule from 'gmp/models/schedule.js';
+import Tag from 'gmp/models/tag.js';
+import Target from 'gmp/models/target.js';
+import Task from 'gmp/models/task.js';
+
+import AlertsTable from '../alerts/table.js';
+import AgentsTable from '../agents/table.js';
+import ScanConfigsTable from '../scanconfigs/table.js';
+import CredentialsTable from '../credentials/table.js';
+import FiltersTable from '../filters/table.js';
+import GroupsTable from '../groups/table.js';
+import NotesTable from '../notes/table.js';
+import OverridesTable from '../overrides/table.js';
+import PermissionsTable from '../permissions/table.js';
+import PortListsTable from '../portlists/table.js';
+import ReportFormatsTable from '../reportformats/table.js';
+import RolesTable from '../roles/table.js';
+import ScannersTable from '../scanners/table.js';
+import SchedulesTable from '../schedules/table.js';
+import TagsTable from '../tags/table.js';
+import TargetsTable from '../targets/table.js';
+import TasksTable from '../tasks/table.js';
+
+import Table from '../../components/table/strippedtable.js';
+import TableBody from '../../components/table/body.js';
+import TableData from '../../components/table/data.js';
+import TableRow from '../../components/table/row.js';
+import TableHead from '../../components/table/head.js';
+import TableHeader from '../../components/table/header.js';
+import {Col} from 'glamorous';
```

Comment:
Please move the glamorous import to the top after the react import.

Reply:
done